Code/GraphMol/Wrap/rough_test.py

```diff
@@ -6248,7 +6248,14 @@ def test_github3553(self):
       txt = sio.getvalue()
       self.assertTrue(pval in txt)
 
-
+  def testMolzip(self):
```

Comment:
Please add one test for the single-argument form of the function too

Reply:
done